conf/summariser.cfg

```diff
@@ -6,3 +6,20 @@ pidfile = /var/run/apel/summariser.pid
 logfile = /var/log/apel/summariser.log
 level = INFO
 console = true
+
+# Define if and how the summariser should handle stale summaries.
+[stale_summaries]
```

Comment:
Doesn't feel like it justifies a separate section with only one setting the the `[summariser]` section. Just needs more descriptive option names.

Reply:
Addressed in [68e12d7](https://github.com/apel/apel/pull/222/commits/68e12d71abc6892e9b4a8ad3b2c79b2426ca068d), but not yet tested.